src/components/ActorList.vue

```diff
@@ -0,0 +1,65 @@
+<template>
+  <div>
+    <div class="d-flex flex-row container" v-if="!loading">
+      <div class="d-flex flex-row">
+          <b-list-group horizontal class="flex-sm-wrap">
+              <b-list-group-item
+                  class="w-auto m-2 p-0 border-0 d-flex justify-content-center align-items-start rounded-circle card"
+                  v-for="actor in actors"
+                  :key="actor.id"
+              >
+                  <actor-card
+                      :name="actor.name"
+                      :knownForDepartment="actor.known_for_department || ''"
+                      :popularity="actor.popularity"
+                      :profilePath="actor.profile_path || ''"
+                      :character="actor.character"
+                      :actorId="actor.id"
+                      @getActorId="getActorDetails"
```

Comment:
Use the `lowercase-hyphen-case` casing for event names.

I.e. rename this event to `get-actor-id` instead of `getActorId`.

All event in JS use that casing, please stick to it going forward.